### PR TITLE
fix shellcheck errors

### PR DESCRIPTION
**Pull request by jn9e9**
Fix for #26 

Signed-off-by: John 9e9 <jn9e9e9@gmail.com>

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.166.1/containers/go/.devcontainer/base.Dockerfile
```
Comment (hug-dev) [inline .devcontainer/Dockerfile:1]: Are the `.devcontainer` files meant to be here or something from your local dev environment?

File: e2etest/scripts/ci.sh [hunk 2]
```diff
@@ -107,44 +92,27 @@ if [ "$PROVIDER_NAME" = "tpm" ] || [ "$PROVIDER_NAME" = "all" ]; then
 fi
 
 if [ "$PROVIDER_NAME" = "pkcs11" ] || [ "$PROVIDER_NAME" = "all" ]; then
-    pushd ${TESTDIR}
+    pushd "${TESTDIR}"
     # This command suppose that the slot created by the container will be the first one that appears
     # when printing all the available slots.
-    SLOT_NUMBER=`softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2`
+    SLOT_NUMBER=$(softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2)
```
Comment (hug-dev) [inline e2etest/scripts/ci.sh:98]: Just FYI, I expect that with https://github.com/parallaxsecond/parsec/issues/375, finding the slot number won't be needed anymore :)

**Review by hug-dev — COMMENTED** [review]
Great 👏 The shellcheck GA might be useful in Parsec too!

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (jn9e9) [reply .devcontainer/Dockerfile]: oops, removed

File: e2etest/scripts/ci.sh (hunk 2 above)
Comment (jn9e9) [reply e2etest/scripts/ci.sh:98]: understood, will remove

**Review by hug-dev — APPROVED** [review]
Perfect 👌

**Review by ionut-arm — APPROVED** [review]
👌🏻

**Pull request merged**